packages/plugin-commands-recursive/src/parsePackageSelectors.ts

```diff
@@ -18,19 +29,31 @@ export default (rawSelector: string, prefix: string): PackageSelector => {
   if (rawSelector.startsWith('...')) {
     const pattern = rawSelector.substring(3)
     return {
+      excludeSelf: false,
+      pattern,
+      scope: 'dependents',
+      selectBy: 'name',
+    }
+  }
+  if (rawSelector.startsWith('..^')) {
```

Comment:
let's make it `...^` because we might use `..` for selecting direct deps only in the future